### DISCUSSION TITLE
LA k-factor priority

### DISCUSSION THREAD

**Discussion by MicroSur** [Jyers/Marlin#1813] [Q&A]
I successfully use the LA function and there is a k-factor option in the advanced menu screen settings. 
The question is what is the priority, this menu setting or the setting in the slicer (Cura)?
I use Deprecated-JyersUI branch source.

**Comment by BlueWings172** [comment]
@MicroSur 

I have seen a guide how to enable Linear Advanced in the Ender 3 V2 and It involves soldering pins on the motherboard.

I'm just wondering if it is worth it. Have you seen considerable quality improvement ?

**Reply by MicroSur** [reply]
I have more accurate geometry on 120mm/s printing speed, but only with direct extruder.
